packages/core/useContainerQuery/index.test.ts

```diff
@@ -0,0 +1,7 @@
+import { useContainerQuery } from '../useContainerQuery'
+
+describe('useContainerQuery', () => {
```

Comment:
should need more test

Reply:
I think as well because of the refactoring, Italy slightly harder to test the breakpoint matching functionality...